lib/vmpooler/pool_manager.rb

```diff
@@ -658,6 +659,21 @@ def get_provider_for_pool(pool_name)
       $providers[provider_name]
     end
 
+    # @param pool_name [String] - the name of the pool
+    # @return [String] - domain name for pool, if set
+    def get_domain_for_pool(pool_name)
+      provider = get_provider_for_pool(pool_name)
```

Comment:
I think this can return `nil` too

Reply:
Thanks, I am adding a new line to bail early if the respone is nil